scripts/build/generateIcons.ts

```diff
@@ -203,7 +203,8 @@ async function loadCodiconMappings(): Promise<Record<string, number | undefined>
     const mappings: Record<string, number | undefined> = {}
     for (const [k, v] of Object.entries(data)) {
         if (typeof k === 'string' && typeof v === 'number') {
-            mappings[`vscode-${k}`] = v - 0xd000
+            // Assumption in the 0XE000 range, verify in mapping.json if needed
```

Comment:
Since this is a build step we should assert our assumptions whenever possible:
```suggestion
            if (v < 0xe000 || v >= 0xf000) {
                throw new Error(`Codicon "${k}" has unexpected codepoint: ${v}`)
            }
```
And then even better would be to dedupe `0xf000` to a new const `startUnicode`. This establishes the connection between the two things.